apps/charterafrica/payload.config.ts

```diff
@@ -33,7 +33,11 @@ dotenv.config({ path: path.resolve(__dirname, "./.env.local") });
 
 const appURL = process.env.PAYLOAD_PUBLIC_APP_URL;
 
-const allowedDomains = process?.env?.PAYLOAD_ALLOWED_DOMAINS?.split(",")
+const allowedCORSDomains = process?.env?.PAYLOAD_CORS_DOMAINS?.split(",")
+  ?.map((d) => d.trim())
+  ?.filter(Boolean);
+
+const allowedCSRFDomains = process?.env?.PAYLOAD_CSRF_DOMAINS?.split(",")
```

Comment:
```suggestion
const cors = process?.env?.PAYLOAD_CORS?.split(",")
  ?.map((d) => d.trim())
  ?.filter(Boolean);

const csrf = process?.env?.PAYLOAD_CSRF?.split(",")
```

I take it these are just URLs and not necessary domains?

Reply:
Yes, these are URLS